index.html

```diff
@@ -163,16 +163,37 @@ <h2>About me</h2>
                 </div>
             </section>
 
-             <div>
+            <section class="form-section">
+             <form action="https://formspree.io/f/mjvzwlqy" method="post">
                 <h2 class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-                <br>
-                <div class="textareas">
-                <textarea name="name" id="" cols="41" rows="2">Full name</textarea>
-                <textarea name="e-mail" id="" cols="41" rows="2">E-mail address</textarea>
-                <textarea name="text" id="" cols="41" rows="10">Enter text here</textarea>
-                <div class="butt"><button class="btn" type="button"> Get in touch</button></div>
-                </div>
-            </div>
+                <ul>
+                    
+                    <li >
+                        <label for="name">
+                            <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                        </label>
+                    </li>
+                    <li>
+                        <label for="mail">
+                            <input type="e-mail" id="mail" name="user_email" placeholder="Email address" required>
+                        </label>
+
```

Comment:
- [ ] Kindly make sure you correct the typo error for the `email input type` as the current one is wrong. 
```HTML
<-- Bad --->
<input type="e-mail" id="mail" />

<-- Good --->
<input type="email" id="mail" />
```